### PR TITLE
Merchant dashboard

### LINKED ISSUE #3: Merchant Dashboard
As a merchant,
When I visit my merchant dashboard (/merchants/merchant_id/dashboard)
Then I see the name of my merchant

### PR DESCRIPTION
**Pull request by thayes87**
Driver/Navigator 
*User stories for merchant dashboard 
*created dashboard model 
*created dashboard feature spec 
*deleted card expiration dates

### CODE REVIEW

**Review by lcole37 — APPROVED** [review]

File: app/views/dashboards/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1>Merchant Dashboard</h1>
+
+<h3><%= @merchant.name %></h3>
+
+<%= link_to "Merchant Items", merchant_items_path(@merchant) %>
```
Comment (Dominicod) [inline app/views/dashboards/index.html.erb:5]: nice use of routes helpers here! Must have been alot of work!

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   resources :transactions
-  resources :merchants
+  resources :merchants do
+    resources :dashboards, only: [:index, :create]
+    resources :items, only: [:index]
+    resources :invoices, only: [:index]
+  end
```
Comment (Dominicod) [inline config/routes.rb:7]: This is crazy how this works, but def super cool.

File: spec/features/merchants/dashboard_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    it 'I see link to my merchant invoices index' do
+      bob = Merchant.create!(name: "Bob's Soaps")
+
+      visit merchant_dashboards_path(bob)
+
+      expect(page).to have_link("Merchant Invoices")
+    end
+  end
+end
```
Comment (Dominicod) [inline spec/features/merchants/dashboard_spec.rb:31]: Prob need to do a sad path

File: spec/features/merchants/dashboard_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      expect(page).to have_content(bob.name)
+      expect(page).to_not have_content(sue.name)
+    end
+
+    it 'I see link to my merchant items index' do
+      bob = Merchant.create!(name: "Bob's Soaps")
+      
+      visit merchant_dashboards_path(bob)
+
+      expect(page).to have_link("Merchant Items")
```
Comment (Dominicod) [inline spec/features/merchants/dashboard_spec.rb:20]: "My Items" instead of "Merchant Items"

File: spec/features/merchants/dashboard_spec.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    end
+
+    it 'I see link to my merchant invoices index' do
+      bob = Merchant.create!(name: "Bob's Soaps")
+
+      visit merchant_dashboards_path(bob)
+
+      expect(page).to have_link("Merchant Invoices")
```
Comment (Dominicod) [inline spec/features/merchants/dashboard_spec.rb:28]: "My Invoices" instead of "Merchant Invoices"

**Review by Dominicod — APPROVED** [review]
This looks great! RTM

**Pull request merged**